src/deploy/hosting/prepare.ts

```diff
@@ -44,6 +44,22 @@ export async function prepare(context: any, options: any): Promise<void> {
       labels: deploymentTool.labels(),
     };
 
+    // Keep a copy of the data in the payload, to patch in later
+    payload.hosting ||= {};
+    payload.hosting.config ||= {};
+    payload.hosting.config[deploy.site] = { ...data.config };
+
+    // Filter out any rewrites that point to GCFv2 functions that are being deployed
+    Object.entries(payload.functions?.backend?.endpoints || {}).forEach(([region, endpoints]) => {
```

Comment:
Strongly prefer `for.. of...` over forEach - go/tsstyle#iterating-containers - applies for both of these loops

Reply:
Ended up ditching both loops, since I could just solve in a single filter pass on the config... 🤦